### ISSUE TITLE
Azure Kinect on Jetson Orin

### ISSUE THREAD

**Issue by omerKurkutlu** [microsoft/Azure-Kinect-Sensor-SDK#1961]
**Azure Kinect - ROS in Nvidia Jetson**

I need to use azure kinect with ros in jetson nano. Currently I use ubuntu 20.04 in Jetson.
Actually I followed this  [link](https://github.com/valdivj/Azure-for-Kinect-Jetson-nano) which is working fine (standalone) when I run the SDK with command ./bin/k4aviewer in the Azure-Kinect_Sensor_SDK/build directory. But I need to use azure kinect with ROS. So then I seach k4a in terminal, I don't see it.

Then, I look for this [website](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/blob/develop/docs/usage.md)(Offical GitHub)
_ARM64 users, please use the same instructions, but use https://packages.microsoft.com/ubuntu/18.04/multiarch/prod for the repository path instead of the default https://packages.microsoft.com/ubuntu/18.04/prod._
But I use ubuntu 20.04. So, I look for https://packages.microsoft.com/ubuntu/20.04/prod which is available but for ARM64 user https://packages.microsoft.com/ubuntu/20.04/multiarch/prod is not there.

So in short I am able to use SDK standalone but not with ROS. I need .../20.04/multiarch/prod or another way to run this SDK in ROS with ./bin/k4aviewer
Also I am open for all suggestions...

**Comment by ShayHQ** [comment]
Hey dude,
You probably won't get the help you need from here (This repository is completely abandoned)

Binaries from ubuntu 18.04 should work on 20.04,  i built my binaries from source.

If you want to execute in headless (without desktop) in the future:
You will need to "mock" desktop environment to populate DISPLAY variable.
